var/spack/repos/builtin/packages/gmsh/package.py

```diff
@@ -29,6 +30,7 @@ class Gmsh(CMakePackage):
 
     variant('shared',      default=True,  description='Enables the build of shared libraries')
     variant('mpi',         default=True,  description='Builds MPI support for parser and solver')
+    variant('openmp',      default=True,  description='Enable OpenMP support')
```

Comment:
please make it `false` by default on macOS.

Reply:
Sure.